### DISCUSSION TITLE
Welcome to Costro Discussions!

### DISCUSSION THREAD

**Discussion by yoriiis** [costrojs/costro#1] [General]
## 👋 Welcome!

We're using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you're wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by aceg82** [comment]
Hello Yoriiis, I am trying your framework right now and when I make the command `npm install costro` all the folders are not like the github structure  🌱

**Reply by yoriiis** [reply]
Hi @aceg82,

Are you referring to the structure of the `node_modules/costro` folder?

If so, this is normal, the folder structure on GitHub is for Costro development. In the node modules folder, it is the contents of the package that you are going to use. The list of declared files is available in [package.json#L44-L46](https://github.com/costrojs/costro/blob/main/package.json#L44-L46) and on [package-json#files on npm](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files).

**Reply by aceg82** [reply]
Hi again,

Yes effectively the github structure is for development. 
I was wondering cause when I `npm install costro` there is only a node_modules folder and a package-lock.json file.

By the way, the light framework is a nice idea.
